projects/components/src/viewer/code-viewer/code-viewer.component.scss

```diff
@@ -68,5 +70,18 @@
         white-space: break-spaces;
       }
     }
+
+    .copy-to-clipboard {
+      position: absolute;
+      color: $gray-9;
+      right: 0;
+      top: 0;
+    }
+  }
+
+  ::ng-deep {
+    mark {
```

Comment:
it won't work without ng-deep? 

also, we should put it inside `.code-lines` class